cumulus/parachains/runtimes/test-utils/src/lib.rs

```diff
@@ -253,6 +254,9 @@ impl<Runtime: BasicParachainRuntime> ExtBuilder<Runtime> {
 			.assimilate_storage(&mut t)
 			.unwrap();
 
+		BasicExternalities::execute_with_storage(&mut t, || {
+			<pallet_session::Pallet<Runtime> as OnGenesis>::on_genesis();
```

Comment:
should we better use `AllPalletsWithSystem as OnGenesis` to also include other usage in the future?